Task19.java

```diff
@@ -0,0 +1,37 @@
+//Создать двухмерный квадратный массив, и заполнить его «бабочкой», т.е
+//таким образом:
+//1 1 1 1 1
+//0 1 1 1 0
+//0 0 1 0 0
+//0 1 1 1 0
+//1 1 1 1 1
+
+package homework3;
+public class Task19 {
+    public static void main(String[] args) {
+        int arr[][] = new int[5][5];
+        int i,j;
+        for (i = 0; i < arr.length/2+1; i++) {
+            for (j = 0; j < arr[i].length; j++) {
+                if ((j < i) || (j >= (arr[i].length - i)))
+                    arr[i][j] = 0;
+                else
+                    arr[i][j] = 1;
+            }
+        }
+        for (i=arr.length-1;i>=arr.length/2+1;i--) {
+            for (j = 0; j < arr[i].length; j++) {
+                if ((j < (arr[i].length - 1 - i)) || (j > i))
+                    arr[i][j] = 0;
+                else
+                    arr[i][j] = 1;
+            }
+        }
+        for (i=0;i<arr.length;i++) {
+            for (j=0;j<arr[i].length;j++) {
+                System.out.print(arr[i][j]);
+            }
+            System.out.println(" ");

```

Comment:
My solution will be something like this:
for (int i = 0; i < array.length; i++) {
for (int j = 0; j < array.length; j++) {
if (i < array.length / 2) {
if (i <= j && array.length - i > j) {
array[i][j] = 1;
}
} else {
if (i >= j && array.length - i < j + 2) {
array[i][j] = 1;
}
}
}
}
please try to analyze. it uses twice less for circles.